client/components/shared/FooterPage.jsx

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import Footer from './Footer';
+
+class FooterPage extends Component {
```

Comment:
this is a redundant component that renders just another component, use Footer in place of FooterePage